cmd/server/assets/codestatus/show.html

```diff
@@ -42,24 +42,32 @@ <h5 class="mb-1">Status</h5>
         <h5 class="mb-1">Expiry</h5>
         <span id="code-expires-at" class="sm text-danger"></span>
       </div>
+      <div class="card-body">
+        <form action="/code/expire" method="POST">
```

Comment:
I wrote some magical javascript that makes this a bit easier. Check out users/index.html for the `delete` action.